aries-site/src/pages/templates/side-to-side-navigation.mdx

```diff
@@ -1,3 +1,89 @@
-import { ComingSoon } from '../../components';
+import { Box, Image, NameValueList, NameValuePair } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When to use
+
+- Tabs to maneuver between relevant information within the same view
```

Comment:
question: should we be putting periods? I added them to my page but to be consistent I wanted to ask

Reply:
yes we should, thank you!